bin/des-pizza-cutter-fit-gaia-mask

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+
+import numpy as np
+import fitsio
+import hickory
+import esutil as eu
+from esutil.numpy_util import between
+
+
+def main():
+    order = 2
+
+    t = fitsio.read('masks.fits', lower=True)
```

Comment:
would be good to document where this file is located so we can find it later

Reply:
it was created by  Robert in some temporary directory of his on some server.  If we want to keep track of it we will need to have Robert make it a part of the official arxiv.  Perhaps better would be to add a script that runs the query

Reply:
sorry "archive"